.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,227 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "5.0.x"
```

Comment:
We need to support all branch pushes, IMO

Reply:
👍